### ISSUE TITLE
Right column on documentation layout doesn't have a scroll

### ISSUE THREAD

**Issue by souzabrs** [apostrophecms/apostrophe-documentation#146]
I don't know if here is the right place, but, the contents right column doesn't have a scroll when the window is maximized on computer without touchscreen, in the way that some items are unreachable.

Like this one: 
https://docs.apostrophecms.org/apostrophe/modules/apostrophe-pieces

I've tested on three different browsers and noticed that the leftmost colum has a attribute `data-scrollable="true"` and the rightmost (contents of the page) don't. I couldn't make the right column scrollable just by adding `overflow-y: scroll` to its CSS, so, maybe it's more complicated.

**Comment by abea** [comment]
You're right. This might be a limitation of Gitbook, but something we can check.
